geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/InfoResultModel.java

```diff
@@ -44,4 +45,9 @@ public InfoResultModel removeLine(int index) {
     messages.remove(index);
     return this;
   }
+
+  @JsonIgnore
+  public String getStringContent() {
```

Comment:
I'd prefer not to add extra methods just to facilitate testing. In general, I'm also not a fan of checking string content if at all possible. I think that:
```
assertThat(result.getInfoSection("info").getStringContent()).contains("foo");
```
Is equivalent to:
```
assertThat(result.getInfoSection("info").getContent()).containsOnly("foo");
```
and
```
assertThat(result.getInfoSection("info").getStringContent()).isEmpty();
```
is just:
```
assertThat(result.getInfoSection("info").getContent()).containsOnly("");
```